js/explore/helpers.js

```diff
@@ -13,3 +13,21 @@ function GetTopX(arrayData, x) {
 	});
 	return arrayData.slice(0, x); 
 };
+
+// Draw date-of-interest reference lines on a graph
+function drawDateLine(dateObj,label,chart,x,y,height,valueline) {
```

Comment:
We might want to add some comments / documentation at some point, but this looks good to me!